packages/grid/x-data-grid-premium/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import './typeOverloads';
+import '@mui/x-data-grid-pro/typeOverloads';
+
+export { LicenseInfo } from '@mui/x-license-pro';
+export * from '@mui/x-data-grid/components';
+export * from '@mui/x-data-grid-pro/components';
+export * from '@mui/x-data-grid/constants';
+export * from '@mui/x-data-grid/hooks';
+export * from '@mui/x-data-grid-pro/hooks';
+export * from '@mui/x-data-grid/locales';
+export * from '@mui/x-data-grid/models';
+export * from '@mui/x-data-grid-pro/models';
+export * from '@mui/x-data-grid/context';
+export * from '@mui/x-data-grid/colDef';
+export * from '@mui/x-data-grid/utils';
+
+export * from './DataGridPremium';
+export * from './hooks';
+export * from './models';
```

Comment:
@flaviendelangle This lines are why I do not export `GridExportFormat`, `GridExportExtension` from `'@mui/x-data-grid/models'`. 

If I do so, it will conflict with the overridden types exported by `export * from './hook` because in `./hook/feature/export` I override types to add excel to the exported formats

Reply:
Make sense :+1: